apps.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:12 as builder
+
+WORKDIR /joystream
+COPY . /joystream
+
+# Do not set NODE_ENV=production until after running yarn install
+# to ensure dev dependencies are installed.
+RUN yarn install --frozen-lockfile
+
+# ENV NODE_ENV=production
+# Pioneer is failing to build only on github actions workflow runner
+# Error: packages/page-staking/src/index.tsx(24,21): error TS2307: Cannot find module './Targets' or its corresponding type declarations.
+# RUN yarn workspace pioneer build
+RUN yarn workspace @joystream/cli build
+RUN yarn workspace storage-node build
+
+# The image is huge ~ 3GB!
```

Comment:
It looks like the reasoning rather than file comments. Do you want to keep it here?

Reply:
I take them out and put them in a new issue

Reply:
clearned in d7cb35bea56883e0a9e5d1cc157dd4a1fa8e41b1
add to issue: https://github.com/Joystream/joystream/issues/1541